text_recognizer/metadata/iam.py

```diff
@@ -6,4 +6,26 @@
 EXTRACTED_DATASET_DIRNAME = DL_DATA_DIRNAME / "iamdb"
 
 DOWNSAMPLE_FACTOR = 2  # If images were downsampled, the regions must also be.
-LINE_REGION_PADDING = 16  # add this many pixels around the exact coordinates
```

Comment:
16 along y-axis was too much padding. It induces noise from the neighboring lines

Reply:
i actually think some noise from other lines is good -- it's extra cruft that's almost never an entire letter, so it changes the input without changing the label. that makes it a form of augmentation!

i'd love to have some augmentation that looks more like "dropping partial ink strokes into the image" to better represent nuisance factors we'll encounter IRL, and this is like a cheap version of that.